include/morphio/vector_types.h

```diff
@@ -21,7 +19,7 @@ constexpr floatType PI = M_PI;
 /** Type of float to use. Can be double or float depending on MORPHIO_USE_DOUBLE */
 using floatType = float;
 /** A really small value that is used to measure how close are two values */
-constexpr floatType epsilon = 1e-6f;
+constexpr floatType epsilon = 1e-6F;
```

Comment:
Should it be upper case, given it's a constant?

Reply:
good point; I'd like to remove it altogether - not a fan of having these sorts of things as constants 'everywhere'.  I'll think about it for another PR.